website/source/docs/guides/consul-splitting.md

```diff
@@ -0,0 +1,461 @@
+
+Next, you’ll create a configuration entry that will split percentages of traffic
+to the subsets of your upstream service that you just defined. Initially, you
+want the splitter to send all traffic to v1 of your upstream service, which
+prevents any traffic from being sent to v2 when you register it. In a production
+scenario, this would give you time to make sure that v2 of your service is up
+and running as expected before sending it any real traffic.
+
+The configuration entry for Service Splitting is of `kind` of
+`service-splitter`. Its `name` specifies which service that the splitter will
+act on. The `splits` field takes an array which defines the different splits; in
+this example, there are only two splits; however, it is [possible to configure
+multiple sequential
+splits](https://www.consul.io/docs/connect/l7-traffic-management.html#splitting).
+
+Each split has a weight which defines the percentage of traffic to distribute to
+each service subset. The total weights for all splits must equal 100. For our
+initial split, we are going to configure all traffic to be directed to the
+service subset v1.
+
+The service splitter already exists in your demo environment at
+`l7_config/api_service_splitter_100_0.json` and contains the following
+configuration.
+
+```json
+{
+  "kind": "service-splitter",
+  "name": "api",
+  "splits": [
+    {
+      "weight": 100,
+      "service_subset": "v1"
+    },
+    {
+      "weight": 0,
+      "service_subset": "v2"
+    }
+  ]
+}
+```
+
+Apply this configuration entry by issuing another PUT request to the Consul’s
+configuration entry endpoint of the HTTP API.
+
+```shell
+$ curl localhost:8500/v1/config -XPUT -d @l7_config/api_service_splitter_100_0.json
+true%
+```
+
+This scenario is the first stage in our Canary deployment; you can now launch
+the new version of your service without it immediately being used by the
+upstream load balancing group.
+
+### Start and Register API Service Version 2
+
+Next you’ll start the canary version of the API service (version 2),  and
+register it with the settings that you used in the configuration entries for
+resolution and splitting. Start the service, register it, and start its connect
+sidecar with the following command. This command will run in the foreground, so
+you’ll need to open a new terminal window after you run it.
+
+```shell
+$ docker-compose -f docker-compose-v2.yml up
+```
+
+Check that the service and its proxy have registered by checking for new `v2`
+tags next to the API service and API sidecar proxies in the Consul UI.
+
+### Configure Service Splitting - 50% Version 1, 50% Version 2
+
+Now that version 2 is running and registered, the next step is to gradually
+increase traffic to it by changing the weight of the v2 service subset in the
+service splitter configuration. Let’s increase the weight of the v2 service to
+50%. Remember; total service weight must equal 100, so you also reduce the
```

Comment:
Can we change the language here and/or above to refer to the weight as a percentage? We explicitly designed this UX as percentages are easier to reason about than arbitrary sums/fractions. It's also important to note that these weights are floating point and can express up to 0.01% resolution. Something like:

> Remember; these are percentages so should sum to 100%, but note that you can use increments as small as 0.01%.

Reply:
Is it also worth acknowledging that in production you'd likely start with a much smaller split until you gained confidence?